src/reducers/ratings.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+/* eslint import/no-cycle: 0 */
+import { handleActions } from "redux-actions";
+import type { State } from ".";
+
+export type RatingsState = {
```

Comment:
It would be good to add a short description above each property of the state.
e.g: 
```
{
  /** here explain what this property is for */
  currentRoute?: string;
}
```